README.md

```diff
@@ -37,15 +37,67 @@ To run the tests:
 docker run --rm multipy multipy/runtime/build/test_deploy
 ```
 
-### Installing `multipy::runtime` from source
+### Installing via `pip install`
+
+We support installing both python modules and the runtime libs using `pip install`, with the caveat of having to manually install the dependencies first.
+
+To start with, the multipy repo should be cloned first:
+```shell
+git clone https://github.com/pytorch/multipy.git
+cd multipy
+git submodule sync && git submodule update --init --recursive
+```
+
+
+#### Installing system dependencies
+
+The runtime system dependencies are specified in `build-requirements.txt`. To install them on Debian-based systems, one could run:
+
+```shell
+sudo apt update
+xargs sudo apt install -y -qq --no-install-recommends <build-requirements.txt
+```
+
+We recommend using the latest version of `cmake` and compilers available for your system. On Ubuntu 18.04, for example, these can be updated as follows:
+
+```shell
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo gpg --dearmor -o /usr/share/keyrings/magic-key.gpg
+echo "deb [arch=amd64,arm64 signed-by=/usr/share/keyrings/magic-key.gpg] https://apt.kitware.com/ubuntu/ bionic main" | sudo tee -a /etc/apt/sources.list
+echo "deb http://security.ubuntu.com/ubuntu focal-security main" | sudo tee -a /etc/apt/sources.list
+sudo apt update
+sudo apt install -y binutils cmake
+```
+
+#### Installing environment encapsulators
+
+We support both `conda` and `pyenv`+`virtualenv` to create isolated environments to build and run in. Since `multipy` requires a position-independent version of python to launch interpreters with, for `conda` environments we use the prebuilt `libpython-static=3.x` libraries from `conda-forge` to link with at build time, and for `virtualenv`/`pyenv` we compile python with `-fPIC` to create the linkable library.
 
-Multipy needs a local copy of python with `-fPIC` enabled as well as a recent copy of pytorch.
+> **NOTE** We support Python versions 3.7 through 3.10 for `multipy`; note that for `conda` environments the `libpython-static` libraries are available for `3.8` onwards. With `virtualenv`/`pyenv` any version from 3.7 through 3.10 can be used, as the PIC library is built explicitly.
 
-#### Dependencies: Conda
+Example commands for installing conda:
+```shell
+curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  && \
+chmod +x ~/miniconda.sh && \
+~/miniconda.sh -b -p /opt/conda && \
+rm ~/miniconda.sh
+```
+
+Virtualenv / pyenv can be installed as follows:
+```shell
+pip3 install virtualenv
+git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+```
```

Comment:
I don't think we need to cover how to install these -- can just link to their install pages. I assume if users are using deploy they're familiar